cypress/cypress/integration/video.spec.ts

```diff
@@ -14,9 +14,12 @@ const FAKE_STYLE_HEADER_LOGO =
 describe("Video Mentor", () => {
   describe("Plays a video in response to a user question", () => {
     it("plays a mentor response and displays subtitles", () => {
-      mockDefaultSetup(cy);
+      mockDefaultSetup(cy, { mentorData: clint });
       cy.visit("/?mentor=clint");
       cy.get("[data-cy=input-field]").type("is the food good");
+      // have to wait until the intro video is done to send a question
+      // to truly check if we are recieving a response to the question
+      cy.wait(8000);
```

Comment:
I'm not sure we can use `wait()`, probably you can add another `data-cy` and update it when after an event. Once it changes, you can use `cy.get("[data-cy=input-field]").type("is the food good").then(() => {
... do something after the event
})`

or just add a `timeout`

Reply:
eabe549